design-system/materials/internals/definition.yaml

```diff
@@ -36,23 +36,33 @@ choiceGroupsByTheme:
         color-primary-40: 'hsl(172.9608938547486, 100%, 40%)'
         color-primary-85: 'hsl(172.9608938547486, 100%, 85%)'
         color-primary-95: 'hsl(172.9608938547486, 100%, 95%)'
+        color-primary-op-90: 'hsl(173deg 100% 35% / 90%)'
+        color-primary-op-95: 'hsl(173deg 100% 35% / 95%)'
```

Comment:
There are some temporary color tokens (the ones with `op`) since I'm not sure how to handle the new requirement about using opacity for buttons background color (5% for hover and 10% for active).

For primary color we already have a `color-primary-95` token which, by name, would fit to be used when a primary button is hovered, however, its value does not seem to be what we expect if a compare it with the base primary color with 95% opacity.

Here is an visual comparision:
![image](https://user-images.githubusercontent.com/97907068/200262785-03ac1f5d-4acb-493b-ba2e-ab8ded73e0c2.png)

Reply:
We had a meeting where we discussed this topic and we decided to not create new low level tokens for the new colors needed but instead use the new values in the decision tokens for the buttons where they are needed.

Updated in this commit: f70abe822d56e2ee755bdefb06605319a94db8ca